src/map/utils/trustutils.cpp

```diff
@@ -186,336 +166,345 @@ namespace trustutils
+void FreeTrustList()
+{
+    g_PTrustIDList.clear();
+}
 
-        float raceStat = 0;         // конечное число HP для уровня на основе расы.
-        float jobStat = 0;          // конечное число HP для уровня на основе первичной профессии.
-        float sJobStat = 0;         // коенчное число HP для уровня на основе вторичной профессии.
-        int32 bonusStat = 0;            // бонусное число HP которое добавляется при соблюдении некоторых условий.
+void LoadTrustStats(CTrustEntity *PTrust)
+{
+    // Cargo cult of PC calculations.
 
-        int32 baseValueColumn = 0;  // номер колонки с базовым количеством HP
-        int32 scaleTo60Column = 1;  // номер колонки с модификатором до 60 уровня
-        int32 scaleOver30Column = 2;    // номер колонки с модификатором после 30 уровня
-        int32 scaleOver60Column = 3;    // номер колонки с модификатором после 60 уровня
-        int32 scaleOver75Column = 4;    // номер колонки с модификатором после 75 уровня
-        int32 scaleOver60 = 2;          // номер колонки с модификатором для расчета MP после 60 уровня
-        int32 scaleOver75 = 3;          // номер колонки с модификатором для расчета Статов после 75-го уровня
+    float raceStat = 0;  // конечное число HP для уровня на основе расы.
+    float jobStat = 0;   // конечное число HP для уровня на основе первичной профессии.
+    float sJobStat = 0;  // коенчное число HP для уровня на основе вторичной профессии.
+    int32 bonusStat = 0; // бонусное число HP которое добавляется при соблюдении некоторых условий.
 
-        uint8 grade;
+    int32 baseValueColumn = 0;   // номер колонки с базовым количеством HP
+    int32 scaleTo60Column = 1;   // номер колонки с модификатором до 60 уровня
+    int32 scaleOver30Column = 2; // номер колонки с модификатором после 30 уровня
+    int32 scaleOver60Column = 3; // номер колонки с модификатором после 60 уровня
+    int32 scaleOver75Column = 4; // номер колонки с модификатором после 75 уровня
+    int32 scaleOver60 = 2;       // номер колонки с модификатором для расчета MP после 60 уровня
+    int32 scaleOver75 = 3;       // номер колонки с модификатором для расчета Статов после 75-го уровня
 
-        uint8 mlvl = PTrust->GetMLevel();
-        uint8 slvl = PTrust->GetSLevel();
-        JOBTYPE mjob = PTrust->GetMJob();
-        JOBTYPE sjob = PTrust->GetSJob();
+    uint8 grade;
 
-        uint8 race = 0;                 //Human
+    uint8 mlvl = PTrust->GetMLevel();
+    uint8 slvl = PTrust->GetSLevel();
+    JOBTYPE mjob = PTrust->GetMJob();
+    JOBTYPE sjob = PTrust->GetSJob();
 
-        switch (PTrust->look.race)
-        {
-        case 3:
-        case 4: race = 1; break;    //Elvaan
-        case 5:
-        case 6: race = 2; break;    //Tarutaru
-        case 7: race = 3; break;    //Mithra
-        case 8: race = 4; break;    //Galka
-        }
+    uint8 race = 0; //Human
 
-        // Расчет прироста HP от main job
+    switch (PTrust->look.race)
```

Comment:
this needs to die, trusts aren't players. there is a trust that is a treant even which would fail here.

Reply:
as an example, a "taru" trust should simply use family ID 153 or 154 (or make a new one if those stats don't match)